mainapp/templates/account/base.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
```

Comment:
It seems this file does nearly nothing. Do we need this file `mainapp/templates/account/base.html`?

Reply:
It actually is necessary because allauth comes with a base.html as well. This base.html overrides their base.html, which messes up our styling.